extensions/resteasy-reactive/quarkus-jaxrs-client/deployment/src/main/java/io/quarkus/resteasy/reactive/client/deployment/ClientEndpointIndexer.java

```diff
@@ -79,9 +79,9 @@ protected InjectableBean scanInjectableBean(ClassInfo currentClassInfo,
 
     protected MethodParameter createMethodParameter(ClassInfo currentClassInfo, ClassInfo actualEndpointInfo, boolean encoded,
             Type paramType, ClientIndexedParam parameterResult, String name, String defaultValue, ParameterType type,
-            String elementType, boolean single) {
+            String elementType, String signature, boolean single) {
```

Comment:
This method signature is a mine field:

- `Type paramType`
- `ParameterType type`
- `String elementType`
- `String signature`

And of course, we reverse the order of `type…signature` to `signature, type` in the call below :(

Reply:
Indeed... I'll move the signature `String` to at least make it less likely to fail